### ISSUE TITLE
NPE in freestyle build

### ISSUE THREAD

**Issue by josb** [jenkinsci/slack-plugin#495]
Slack plugin 2.14, Jenkins 2.150.2; let me know if you need any other environment values.

Seeing this NPE in a freestyle job:
```
FATAL: null
java.lang.NullPointerException
	at java.util.regex.Matcher.getTextLength(Matcher.java:1283)
	at java.util.regex.Matcher.reset(Matcher.java:309)
	at java.util.regex.Matcher.<init>(Matcher.java:229)
	at java.util.regex.Pattern.matcher(Pattern.java:1093)
	at jenkins.plugins.slack.ActiveNotifier$MessageBuilder.extractReplaceLinks(ActiveNotifier.java:508)
	at jenkins.plugins.slack.ActiveNotifier$MessageBuilder.escape(ActiveNotifier.java:526)
	at jenkins.plugins.slack.ActiveNotifier$MessageBuilder.append(ActiveNotifier.java:382)
	at jenkins.plugins.slack.ActiveNotifier.started(ActiveNotifier.java:69)
	at jenkins.plugins.slack.SlackNotifier.prebuild(SlackNotifier.java:464)
	at hudson.model.AbstractBuild$AbstractBuildExecution.preBuild(AbstractBuild.java:798)
	at hudson.model.AbstractBuild$AbstractBuildExecution.preBuild(AbstractBuild.java:793)
	at hudson.model.Build$BuildExecution.doRun(Build.java:145)
	at hudson.model.AbstractBuild$AbstractBuildExecution.run(AbstractBuild.java:504)
	at hudson.model.Run.execute(Run.java:1810)
	at hudson.model.FreeStyleBuild.run(FreeStyleBuild.java:43)
	at hudson.model.ResourceController.execute(ResourceController.java:97)
	at hudson.model.Executor.run(Executor.java:429)
```

**Comment by timja** [comment]
Duplicate of https://github.com/jenkinsci/slack-plugin/issues/479
Can you add your config to that issue please, I'll see if I can reproduce it soon, but it seems environmental specific as you're only the second person to report this 😢

**Comment by timja** [comment]
Released as 2.15, may take some time for it to be available in the update centre
